locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po

```diff
@@ -736,22 +736,30 @@ msgstr ""
 "экземпляра Tarantool'а называется 'interactive', а новые файберы, "
 "созданные с помощью :ref:`fiber.create <fiber-create>`, называются 'lua'."
 " Переименование файберов позволяет легче различать их при использовании "
-":ref:`fiber.info <fiber-info>`."
+":ref:`fiber.info <fiber-info>`. Максимум 32 символа."
 
 msgid "the new name of the fiber."
 msgstr "новое имя файбера."
 
+#, fuzzy
```

Comment:
if you validated the translation, please remove the fuzzy flag here

Reply:
fixed in 9f63d45